libraries/testkit/syncgateway.py

```diff
@@ -23,6 +24,10 @@ def __init__(self, cluster_config, target):
         self._headers = {'Content-Type': 'application/json'}
         self.admin = Admin(self)
 
+        self.cluster_config = cluster_config
+        self.server_port = 8091
+        self.server_scheme = "http"
```

Comment:
When initializing the class, why not set a bool `server_ssl`. Then below in .start(), just set the port and scheme as local variables. That way you can reduce the amount of state on the class.